spring-cloud-stream-binder-kinesis-docs/src/main/asciidoc/overview.adoc

```diff
@@ -155,4 +155,15 @@ sendTimeout::
 Default: `10000`
 
 
+[[kinesis-error-channels]]
+== Error Channels
+
+The binder can be configured to send producer exceptions to an error channel. See <<binder-error-channels>> for more information.
```

Comment:
What is this `binder-error-channels` internal link about?
I guess you mean something like this: https://docs.spring.io/spring-cloud-stream/docs/Ditmars.RELEASE/reference/htmlsingle/#binder-error-channels. So, let's make the real URL here, not an internal for nothing!